docs/en/pact-functions.rst

```diff
@@ -158,6 +158,17 @@ Test that LIST or STRING contains VALUE, or that OBJECT has KEY entry.
    pact> (contains "foo" "foobar")
    true
 
+continue
+~~~~~~~~
+
+*value* ``*`` *→* ``*``
+
+Continue a previously started nested defpact.
+
+.. code:: lisp
+
+   (continue f)
```

Comment:
```suggestion
   (continue (coin.transfer-crosschain "bob" "alice" 10.0)
```

This is for the haskell code obviously. Confirm that it is safe to upgrade examples, but I believe it is.